src/axolotl/monkeypatch/llama_attn_hijack_flash.py

```diff
@@ -44,6 +49,16 @@ def replace_llama_attn_with_flash_attn(packed: Optional[bool] = False):
             llama_model_forward
         )
 
+    try:
+        from flash_attn.losses.cross_entropy import CrossEntropyLoss
+
+        LOG.info("patching with flash_attn.losses.cross_entropy")
+        transformers.models.llama.modeling_llama.CrossEntropyLoss = partial(
+            CrossEntropyLoss, inplace_backward=True
+        )
+    except ImportError:
+        pass
```

Comment:
```suggestion
        LOG.warning("unable to import flash_attn.losses.cross_entropy.CrossEntropyLoss")
```

Reply:
since most users won't have this installed, i wasn't sure about using a warning. but i agree it makes sense to log something.

```suggestion
        LOG.info("optimized flash-attention CrossEntropyLoss not found (run `pip install git+https://github.com/Dao-AILab/flash-attention.git#egg=xentropy_cuda_lib&subdirectory=csrc/xentropy`)"
```